subprojects/language-java/src/integTest/groovy/org/gradle/java/compile/incremental/JavaClassChangeIncrementalCompilationIntegrationTest.groovy

```diff
@@ -508,7 +542,6 @@ class JavaClassChangeIncrementalCompilationIntegrationTest extends BaseJavaClass
             class A {
                 public void method() {
                     List<Deleted> l = new ArrayList<Deleted>();
-                    System.out.println(l.hashCode()); 
```

Comment:
Do you know why this code has been in here?

Reply:
I believe this is leftover from some debugging. Test checks that we detect `List<Deleted>` for local variable. Right @jbartok?